Backend/src/main/java/com/abhinternship/auctionapp/service/UserService.java

```diff
@@ -21,27 +23,28 @@
     private BCryptPasswordEncoder bCryptPasswordEncoder;
 
     @Override
-    public User getById(Integer requestId) {
+    public User getById(Long requestId) {
+        //TODO
         return null;
     }
 
-    public User checkEmail(String email){
+    public boolean checkEmail(String email) {
         User emailDb = repository.findByEmail(email);
```

Comment:
no need to get the entire user model, you can just use an exists query that will return boolean